Dockerfile

```diff
@@ -7,7 +7,26 @@ ADD . .
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o deck
 
 FROM alpine:3.10
-RUN apk --no-cache add ca-certificates
-WORKDIR /root/
+RUN apk update \
+    && apk upgrade \
+    && apk --no-cache add ca-certificates
+
+ARG USER=deck
+ARG GROUP=deck
+ARG UID=9999
+ARG GID=9999
```

Comment:
This allows you to set those variables from outside during the build time of the container @digikin

Reply:
I don't think we really had to template these in the first place. This is a self-contained and small Dockerfile.

No strong feelings here so not changes necessary.